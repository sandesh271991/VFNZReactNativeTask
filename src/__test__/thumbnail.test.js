import React, { Component } from 'react';
import { shallow } from 'enzyme';
import expect from 'expect';
import { Provider } from 'react-redux';
import __Thumbnail from '../component/ThumbnailComponent/thumbnailView';

describe('Thumbnail Screen', () => {
  it('should render Thumbnail component', () => {
    const wrapper = shallow(
        <Provider store={store}>
        <__Thumbnail />
    </Provider>,
    );
  });


  it('should render initial layout', () => {
    // when
    const component = shallow(<Provider store={store}>
        <__Thumbnail />
    </Provider>);
      // then
    expect(component.getElements()).toMatchSnapshot();
  });
});
