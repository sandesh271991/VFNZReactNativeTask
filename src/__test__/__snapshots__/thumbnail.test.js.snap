// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testpage Component should render initial layout 1`] = `
Array [
  <Context.Provider
    value={
      Object {
        "store": Object {
          "clearActions": [Function],
          "dispatch": [Function],
          "getActions": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
        },
        "subscription": Subscription {
          "handleChangeWrapper": [Function],
          "listeners": Object {
            "notify": [Function],
          },
          "onStateChange": [Function],
          "parentSub": undefined,
          "store": Object {
            "clearActions": [Function],
            "dispatch": [Function],
            "getActions": [Function],
            "getState": [Function],
            "replaceReducer": [Function],
            "subscribe": [Function],
          },
          "unsubscribe": null,
        },
      }
    }
  >
    <Memo(Connect(Thumbnail)) />
  </Context.Provider>,
]
`;

exports[`Thumbnail Screen should render initial layout 1`] = `
Array [
  <Context.Provider
    value={
      Object {
        "store": Object {
          "clearActions": [Function],
          "dispatch": [Function],
          "getActions": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
        },
        "subscription": Subscription {
          "handleChangeWrapper": [Function],
          "listeners": Object {
            "notify": [Function],
          },
          "onStateChange": [Function],
          "parentSub": undefined,
          "store": Object {
            "clearActions": [Function],
            "dispatch": [Function],
            "getActions": [Function],
            "getState": [Function],
            "replaceReducer": [Function],
            "subscribe": [Function],
          },
          "unsubscribe": null,
        },
      }
    }
  >
    <Memo(Connect(Thumbnail)) />
  </Context.Provider>,
]
`;
